Wcs compass (#15434)
astropy/astropy: docs/visualization/wcsaxes/overlaying_coordinate_systems.rst

    overlay['dec'].set_ticklabel(color='magenta', size=6)
    overlay['dec'].set_axislabel_position('t')
    overlay['dec'].set_axislabel('Declination', color='magenta')

> ## Overlaying a Compass Arrow
> *****************************

Reviewer: No need to mix Markdown and RST. 😸 
```suggestion
Overlaying a Compass Arrow
**************************
```

    overlay['dec'].set_ticklabel(color='magenta', size=6)
    overlay['dec'].set_axislabel_position('t')
    overlay['dec'].set_axislabel('Declination', color='magenta')

Overlaying a Compass Arrow
**************************

It is often useful to add compass arrows to your images, denoting which directions correspond to North and East on the sky.

> The function :meth:`~astropy.wcs.utils.north_pole_angle` calculates the correct angle for this compass, which can easily be displayed using a matplotlib :class:`~matplotlib.mpl_toolkits.axes_grid1.anchored_artistsAnchoredDirectionArrows()` artist.

Reviewer: Break off long lines. Also typo?

```suggestion
The function :meth:`~astropy.wcs.utils.north_pole_angle` calculates the correct angle for this compass,
which can easily be displayed using a matplotlib :class:`~matplotlib.mpl_toolkits.axes_grid1.anchored_artists.AnchoredDirectionArrows` artist.
```

> 

Reviewer: ```suggestion
```

> .. plot::

Reviewer: Hard for me to tell what this is supposed to show me unless the RTD build is fixed. Would be nice to actually overlay this on `imshow` since I think that is how this compass is meant to be used?
